.github/workflows/buildPlugin.yml

```diff
@@ -98,13 +96,20 @@ jobs:
     
     # Check out the repository in the pull request
     - name: Checkout repository
-      uses: actions/checkout@v4
+      uses: actions/checkout@main
       with:
         ref: ${{ github.head_ref }}
-            
-    # Install plugin from the pull request using the Scipion3 installp command
+      
     - name: Install plugin from pull request
       working-directory: ${{ github.workspace }}
+      run: ../scipion/scipion3 installp -p . --devel
+    
+    # Currently, test script is being downloaded from scipion-chem, but that should be temporary
```

Comment:
Maybe add a TODO flag, so that this issue can be tracked (at the expense of a sonar cloud warning)

Reply:
I'm not sure if SonarCloud tracks .yml files for code smells. However, I can add a TODO, but that does not deppend on me, since having a standalone repository for this single file is completely overkill, and it's integration within one of the scipion packages (scipion-app probably) is not up to me, so, it would be a TODO for someone else...maybe?

Reply:
Okay, I understand it